CITATION.cff

```diff
@@ -45,6 +45,6 @@ authors:
 - family-names: Theisen
   given-names: Merel
 title: Kedro
-version: 0.18.10
-date-released: 2023-06-08
+version: 0.18.11
+date-released: 2023-06-29
```

Comment:
```suggestion
date-released: 2023-06-30
```
😢

Reply:
Will update to `2023-07-03` instead - probably best to avoid a Friday afternoon release 😬